brain/compartmentReportView.cpp

```diff
@@ -99,6 +99,54 @@ std::future<brion::Frames> CompartmentReportView::load(double start, double end)
     return _impl->report->loadFrames(start, end);
 }
 
+std::future<brion::Frames> CompartmentReportView::load(double start, double end,
+                                                       const double step)
+{
+    const double reportTimeStep = _impl->report->getTimestep();
+    const double reportStartTime = _impl->report->getStartTime();
+
+    if (end <= start)
+        throw std::logic_error("Invalid interval");
+
+    if (step < reportTimeStep || step <= 0.)
+        throw std::logic_error("Invalid step");
+
+    // check step is multiple of timestep
+    if (fmod(step, reportTimeStep) != 0.0)
+        throw std::logic_error(
+            "Step should be a multiple of the report time step");
+
+    /// check with Juan
```

Comment:
Comment: Making sure the timestamps we are going to request always fall in the middle of a frame. For that we snap start to the beginning of the frame it's contained and then we add half the time step.